src/test/scala/zio/sqs/ZioSqsSpec.scala

```diff
@@ -0,0 +1,201 @@
+package zio.sqs
+
+import java.net.URI
+
+import org.elasticmq.rest.sqs.SQSRestServerBuilder
+import org.scalatest.{ FlatSpec, Matchers }
+import scalaz.zio.{ DefaultRuntime, Task }
+import software.amazon.awssdk.auth.credentials.{ AwsBasicCredentials, StaticCredentialsProvider }
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.sqs.SqsAsyncClient
+import software.amazon.awssdk.services.sqs.model.Message
+
+class ZioSqsSpec extends FlatSpec with Matchers {
+  "ZioSqsSpec" should "send a message" in {
+    val result = for {
+      server <- Task(
+                 SQSRestServerBuilder.start()
+               )
+      client <- Task {
+                 SqsAsyncClient
+                   .builder()
+                   .region(Region.AP_NORTHEAST_2)
+                   .credentialsProvider(
+                     StaticCredentialsProvider.create(AwsBasicCredentials.create("key", "key"))
+                   )
+                   .endpointOverride(new URI("http://localhost:9324"))
+                   .build()
+               }
+      queueName = "TestQueue"
+      _         <- Utils.createQueue(client, queueName)
+      queueUrl  <- Utils.getQueueUrl(client, queueName)
+      _         <- SqsPublisher.send(client, queueUrl, "hello")
+      message <- SqsStream(
+                  client,
+                  queueUrl,
+                  SqsStreamSettings(stopWhenQueueEmpty = true, waitTimeSeconds = 3)
+                ).map(_.body()).run(scalaz.zio.stream.Sink.await[String])
+      _ <- Task(server.stopAndWait())
+    } yield {
+      message shouldBe "hello"
+    }
+
+    val runtime = new DefaultRuntime {}
```

Comment:
If you make `ZioSqsSpec` extend `DefaultRuntime`, you can avoid this line and call `unsafeRun` directly.